hr_holidays_imposed_days/tests/test_holidays_imposed_days.py

```diff
@@ -148,15 +148,16 @@ def test_imposed_employee_create_validate(self):
              'date_to': fields.Datetime.to_string(self.today +
                                                   relativedelta(days=4)),
              'status_id': self.holiday_type.id,
-             'auto_confirm': True
+             'auto_confirm': True,
+             'employee_ids': [(6, 0, [self.employee2.id])]
              }
         )
         self.imposed.validate()
         leaves = self.holiday_model.search(
             [('type', '=', 'remove'),
              ('employee_id', 'in', (self.employee.id, self.employee2.id))]
         )
-        self.assertEqual(len(leaves), 2)
+        self.assertEqual(len(leaves), 1)
```

Comment:
Why this number is reduced after change?

Reply:
@moylop260 Because of we are Added an employee_ids in the self.imposed record, and after search employee for leaves. If search self.employee2 then leave is 1, therefore reduced leaves after a change.